### PR TITLE
fix(SSR): isInputTypeSupported() and use of 'document' breaks TORA build

### PR DESCRIPTION
**Pull request by fpurcell**
this hides a use of 'document' from our SSR / yarn build, which otherwise complains about the use of an unknown (server-side) variable.

### CODE REVIEW

File: packages/trip-form/src/DateTimeSelector/index.tsx [hunk 1]
```diff
@@ -89,9 +89,17 @@ interface DepartArriveOption {
  * @param {*} type One of the HTML5 input types.
  */
 function isInputTypeSupported(type: string): boolean {
-  const input = document.createElement("input");
-  input.setAttribute("type", type);
-  return input.type === type;
+  let retVal = false;
+  try {
```
Comment (binh-dam-ibigroup) [inline packages/trip-form/src/DateTimeSelector/index.tsx:93]: If `typeof document !== "undefined"` is the differentiator between server-side rendering and browser rendering, then is the try/catch block necessary?

Comment (fpurcell) [reply packages/trip-form/src/DateTimeSelector/index.tsx:93]: You're right Binh .. it was a bit extra, and latest push has removed it.

Comment (binh-dam-ibigroup) [reply packages/trip-form/src/DateTimeSelector/index.tsx]: Cool! With the latest changes, you should be able to replace `retVal` with return statements.

**Review by binh-dam-ibigroup — APPROVED** [review]
LGTM. Inlining `retVal` would be nice.

File: packages/trip-form/src/DateTimeSelector/index.tsx [hunk 2]
```diff
@@ -89,9 +89,13 @@ interface DepartArriveOption {
  * @param {*} type One of the HTML5 input types.
  */
 function isInputTypeSupported(type: string): boolean {
-  const input = document.createElement("input");
-  input.setAttribute("type", type);
-  return input.type === type;
+  let retVal = false;
+  if (typeof document !== "undefined") {
```
Comment (binh-dam-ibigroup) [inline packages/trip-form/src/DateTimeSelector/index.tsx:93]: @fpurcell Could you also please insert a comment explaining that the check is for server-side rendering.

**Review by miles-grant-ibigroup — APPROVED** [review]
Looks good to me. Glad the fix is this easy!

**Pull request merged**